components/sendImageForm.tsx

```diff
@@ -0,0 +1,88 @@
+'use client'
+
+import type React from 'react'
+import { useRef, useState } from 'react'
+import { ImageInput } from './imageInput'
+
+const IMAGE_ID = 'image'
```

Comment:
```suggestion
// TODO: create and assign unique id
const IMAGE_ID = 'image'
```
I think `IMAGE_ID` should be only one.
However, I have no idea to make it.
I'll recommend to add comment at that time.

Reply:
ok.